### ISSUE TITLE
Add a pre-compiled build for lazy people

### ISSUE THREAD

**Issue by Biggusjohnnus** [daniele-rapagnani/alephone-android#8]
Man, I dunno a damn darn thing about java, code & compiling, I just want to play the game.

**Comment by daniele-rapagnani** [comment]
Hey, I understand but there's still some work left to be done to make a ready to run pre-compiled APK. I haven't had much time to work on this lately but it's nice to know someone is interested, I'll keep this issue updated.

**Comment by CptKalibak** [comment]
Hi, I just wanted to let you know. Biggusjohnnus is not the only one. Same here. Im searching for a finished apk with which I can just start playing the Marathon series!
Great Work if it works btw!

**Comment by daniele-rapagnani** [comment]
Hey! Thanks for the interest. I'll do it eventually, I've just been working on other stuff lately.

**Comment by tcely** [comment]
I took a try at building this.

```
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':hidapi'.
> Gradle project ndkBuild.path is /__w/alephone-android/alephone-android/SDL2/src/main/cpp/SDL/src/hidapi/android/jni/Android.mk but that file doesn't exist
```

If you could point me towards what I need to do to fix this error, that would help.

**Comment by daniele-rapagnani** [comment]
> I took a try at building this.
> 
> ```
> FAILURE: Build failed with an exception.
> 
> * What went wrong:
> A problem occurred configuring project ':hidapi'.
> > Gradle project ndkBuild.path is /__w/alephone-android/alephone-android/SDL2/src/main/cpp/SDL/src/hidapi/android/jni/Android.mk but that file doesn't exist
> ```
> 
> If you could point me towards what I need to do to fix this error, that would help.
> 
> Here is the GitHub action I'm working with:
> https://github.com/tcely/alephone-android/blob/master/.github/workflows/gradle.yml

I'm not familiar with GitHub's actions but is it possible it is simply not initializing submodules? (Thus not finding SDL2)

**Comment by tcely** [comment]
> I'm not familiar with GitHub's actions but is it possible it is simply not initializing submodules? (Thus not finding SDL2)

It's something to look into. Thanks!

**Comment by tarmacprefecture** [comment]
Hey, been on a Marathon kick again and was hoping to get this running on my phone. I, like the OP, am illiterate with the ways of compiling and other sorceries. Should I just do some research or is there an APK kicking around here somewhere? Regardless, thanks for doing God's work. Now beam me outa here!

**Comment by daniele-rapagnani** [comment]
> Hey, been on a Marathon kick again and was hoping to get this running on my phone. I, like the OP, am illiterate with the ways of compiling and other sorceries. Should I just do some research or is there an APK kicking around here somewhere? Regardless, thanks for doing God's work. Now beam me outa here!

Hey there! You are in luck, I've resumed working on the port after months and I am working on this very issue to distribute automatic builds of the three chapters in APK form. I have the system almost working but the builds inexplicably crash on startup even if they work great if I build them locally on my PC. I'm working on the issue but if I'm not able to fix it in the following days I'll build the game locally and release it anyway.

TL;DR: There are no APKs at the moment but I'll upload them shortly (probably days)

**Comment by CLOCKW3RK** [comment]
That's really cool. I've been working on a project to make icons for most
windows emulators coded in batch/vbs.
But I'll have to check out your project soon. Thanks fir the update.

**Comment by tarmacprefecture** [comment]
Awesome, thank you for the reply and the work you have already done and continue to do. I should really start looking into java basics since the last coding I did was some python in high school and I do jack shit right now aside from doodling and playing guitar. I'll be eagerly awaiting an update when you can get around to it.

**Comment by daniele-rapagnani** [comment]
Hey guys, I finally made it. You can find all three chapters in the [releases](https://github.com/daniele-rapagnani/alephone-android/releases) page. There some issues, mainly the game taking a lot to boot (almost a minute). Take a look at the README file for more details.

Keeping the APKs updated should be almost automatic now, so you should always be able to download the latest release from now on.
